edgelet/version.txt

```diff
@@ -1 +1 @@
-1.0.10.2
\ No newline at end of file
+1.0.10.3
```

Comment:
Is the edgelet version changing? I don't see any edgelet updates in the changelog...